contents/tutorials/product-data-in-new-tab.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Adding product data in new tab with Momentum Dash
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-11-01
+featuredImage: ../images/tutorials/banners/product-data-in-new-tab.png
+topics: ['apps']
+---
+
+Keeping product analytics top of mind keeps you focused on what matters to your product and customers. It reminds you about the state of the product, and what you need to do to improve it.
+
+When you spend lots of time on the computer, one of the areas you see the most is the new tab page in your browser. There are many extensions for changing and customizing the new tab page, but one of the most popular (and one of our favorites) is [Momentum Dash](https://momentumdash.com/). It transforms the new tab into “a focused, productive, and inspiring dashboard.” 
+
+In this tutorial, we’re going add key product data from PostHog into Momentum Dash’s new tab dashboard. We’ll show you how to format the API request that Momentum Dash uses to get the information from PostHog. To do this, you need a PostHog instance and a [Momentum Dash Plus](https://momentumdash.com/plus) account.
+
+## Getting the information for the API request in PostHog
+
+To get our PostHog data into Momentum Dash, we need to format an API request that “gets” that metric. Momentum Dash makes this API request every time you open a new tab to fill in the key metric information. We are going to need 3 pieces of information from PostHog to create an API request.
+
+### 1. A personal (or project) key
+
+We recommend creating a personal API key in PostHog for the API request. To do so, click your account icon in the top right corner, and then the gear next to your account name. In “My settings,” scroll down to “Personal API Keys” and click “Create personal API key.” Name it something like “Momentum.” Once created, make sure you copy and save it because you must create a new one if you lose it.
+
+![Personal API key](../images/tutorials/product-data-in-new-tab/personal-api-key.png)
+
+You can also use the project key (not recommended). You find it in “Project settings” under “Project Variables.”
+
+### 2. A project ID
+
+You find your project key by clicking “Project settings” in the sidebar, then scrolling down to the “Project Variables” and copying the value for “Project ID.” It should be a series of digits. 
```

Comment:
```suggestion
Alternatively, you can use the project key, but we don't recommend this because it cannot be revoked. 

If you do want to use it you can find your project key by clicking “Project settings” in the sidebar, then scrolling down to the “Project Variables” and copying the value for “Project ID.” It should be a series of digits. 
```

Reply:
Oops, this was a typo. We need project ID (which isn't as important).